.gitignore

```diff
@@ -52,6 +52,8 @@ yarn-error.log
 # via build.py and should not be committed to the repo.
 build/*
 
+# .python-version is generated dynamically so needs to be ignored.
+.python-version
```

Comment:
Are we expecting this `.python-version` to exist in the `../oppia` dir and not in `./oppia` dir? (not sure!)

@vojtechjelinek Can you confirm whether it's fine to add this to .gitignore?

Reply:
It is probably better to have it in the `..` folder, but I think it is still fine to put this file here.